### DISCUSSION TITLE
Italic font for comments

### DISCUSSION THREAD

**Discussion by redeemefy** [zed-industries/zed#6613] [Feature Requests]
Is there a way to make font italic for commented out code?

**Comment by widersky** [comment]
AFAIK not yet.

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
